test-support/helpers/ember-test-utils/describe-component.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Shortcuts for generating the params passed to describeComponent from ember-mocha
+ */
+
+import Ember from 'ember'
+import _ from 'lodash'
+
+/**
+ * A shortcut for filling in the first three params in a describeComponent
+ * @param {String} name - the name of the component
+ * @param {Object} options - any additional options to set
+ * @returns {Object[]} an array of items that need to be passed in to describeComponent
+ */
+function testCtx (name, options = {}) {
+  const testType = (options.unit) ? 'Unit' : 'Integration'
+  return [
+    name,
+    `${testType}: ${Ember.String.classify(name)}Component`,
+    options
+  ]
+}
+
+/**
+ * A shortcut for filling in the text context in a describeComponent
```

Comment:
Good question, an arbitrary name for the first three parameters to the `describeComponent`